cmd/metrics.go

```diff
@@ -38,6 +40,13 @@ var (
 		},
 		[]string{"api"},
 	)
+	httpReplicationDuration = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "s3_ttfb_seconds",
```

Comment:
Why is it called `ttfb`? It doesn't seem to contain time to first byte.

Reply:
It's actually s3_duration_seconds

Reply:
oops copy/paste problem